torchvision/prototype/models/regnet.py

```diff
@@ -75,7 +75,17 @@ class RegNet_Y_400MF_Weights(WeightsEnum):
             "acc@5": 91.716,
         },
     )
-    default = ImageNet1K_V1
+    ImageNet1K_V2 = Weights(
+        url="https://download.pytorch.org/models/regnet_y_400mf-e6988f5f.pth",
+        transforms=partial(ImageNetEval, crop_size=224, resize_size=232),
+        meta={
+            **_COMMON_META,
+            "recipe": "https://github.com/pytorch/vision/issues/3995#new-recipe",
+            "acc@1": 75.804,
```

Comment:
Could you confirm that all validation metrics are estimated with batch-size 1?

Reply:
Yes, verified all the weights with the old API:
```bash
cd /data/home/kaizh/repos/vision/references/classification
srun -p dev --cpus-per-task=16 -t 5:00:00 --gpus-per-node=2 torchrun --nproc_per_node=1 train.py --model regnet_x_400mf --pretrained --test-only --val-resize-size 232 -b 1 | tee ~/logs/verify_regnet_x_400mf.out | grep "Test:  Acc@1"
srun -p dev --cpus-per-task=16 -t 5:00:00 --gpus-per-node=2 torchrun --nproc_per_node=1 train.py --model regnet_x_800mf --pretrained --test-only --val-resize-size 232 -b 1 | tee ~/logs/verify_regnet_x_800mf.out | grep "Test:  Acc@1"
srun -p dev --cpus-per-task=16 -t 5:00:00 --gpus-per-node=2 torchrun --nproc_per_node=1 train.py --model regnet_x_1_6gf --pretrained --test-only --val-resize-size 232 -b 1 | tee ~/logs/verify_regnet_x_1_6gf.out | grep "Test:  Acc@1"
srun -p dev --cpus-per-task=16 -t 5:00:00 --gpus-per-node=2 torchrun --nproc_per_node=1 train.py --model regnet_x_3_2gf --pretrained --test-only --val-resize-size 232 -b 1 | tee ~/logs/verify_regnet_x_3_2gf.out | grep "Test:  Acc@1"
srun -p dev --cpus-per-task=16 -t 5:00:00 --gpus-per-node=2 torchrun --nproc_per_node=1 train.py --model regnet_x_8gf --pretrained --test-only --val-resize-size 232 -b 1 | tee ~/logs/verify_regnet_x_8gf.out | grep "Test:  Acc@1"
srun -p dev --cpus-per-task=16 -t 5:00:00 --gpus-per-node=2 torchrun --nproc_per_node=1 train.py --model regnet_x_16gf --pretrained --test-only --val-resize-size 232 -b 1 | tee ~/logs/verify_regnet_x_16gf.out | grep "Test:  Acc@1"
srun -p dev --cpus-per-task=16 -t 5:00:00 --gpus-per-node=2 torchrun --nproc_per_node=1 train.py --model regnet_x_32gf --pretrained --test-only --val-resize-size 232 -b 1 | tee ~/logs/verify_regnet_x_32gf.out | grep "Test:  Acc@1"
srun -p dev --cpus-per-task=16 -t 5:00:00 --gpus-per-node=2 torchrun --nproc_per_node=1 train.py --model regnet_y_400mf --pretrained --test-only --val-resize-size 232 -b 1 | tee ~/logs/verify_regnet_y_400mf.out | grep "Test:  Acc@1"
srun -p dev --cpus-per-task=16 -t 5:00:00 --gpus-per-node=2 torchrun --nproc_per_node=1 train.py --model regnet_y_800mf --pretrained --test-only --val-resize-size 232 -b 1 | tee ~/logs/verify_regnet_y_800mf.out | grep "Test:  Acc@1"
srun -p dev --cpus-per-task=16 -t 5:00:00 --gpus-per-node=2 torchrun --nproc_per_node=1 train.py --model regnet_y_1_6gf --pretrained --test-only --val-resize-size 232 -b 1 | tee ~/logs/verify_regnet_y_1_6gf.out | grep "Test:  Acc@1"
srun -p dev --cpus-per-task=16 -t 5:00:00 --gpus-per-node=2 torchrun --nproc_per_node=1 train.py --model regnet_y_3_2gf --pretrained --test-only --val-resize-size 232 -b 1 | tee ~/logs/verify_regnet_y_3_2gf.out | grep "Test:  Acc@1"
srun -p dev --cpus-per-task=16 -t 5:00:00 --gpus-per-node=2 torchrun --nproc_per_node=1 train.py --model regnet_y_8gf --pretrained --test-only --val-resize-size 232 -b 1 | tee ~/logs/verify_regnet_y_8gf.out | grep "Test:  Acc@1"
srun -p dev --cpus-per-task=16 -t 5:00:00 --gpus-per-node=2 torchrun --nproc_per_node=1 train.py --model regnet_y_16gf --pretrained --test-only --val-resize-size 232 -b 1 | tee ~/logs/verify_regnet_y_16gf.out | grep "Test:  Acc@1"
srun -p dev --cpus-per-task=16 -t 5:00:00 --gpus-per-node=2 torchrun --nproc_per_node=1 train.py --model regnet_y_32gf --pretrained --test-only --val-resize-size 232 -b 1 | tee ~/logs/verify_regnet_y_32gf.out | grep "Test:  Acc@1"
```

Reply:
```bash
tail -n 2 ~/logs/verify_regnet_*
==> /data/home/kaizh/logs/verify_regnet_x_16gf.out <==
Test:  Total time: 0:21:04
Test:  Acc@1 82.716 Acc@5 96.196

==> /data/home/kaizh/logs/verify_regnet_x_1_6gf.out <==
Test:  Total time: 0:10:34
Test:  Acc@1 79.668 Acc@5 94.922

==> /data/home/kaizh/logs/verify_regnet_x_32gf.out <==
Test:  Total time: 0:21:04
Test:  Acc@1 83.014 Acc@5 96.288

==> /data/home/kaizh/logs/verify_regnet_x_3_2gf.out <==
Test:  Total time: 0:18:13
Test:  Acc@1 81.196 Acc@5 95.430

==> /data/home/kaizh/logs/verify_regnet_x_400mf.out <==
Test:  Total time: 0:11:31
Test:  Acc@1 74.864 Acc@5 92.322

==> /data/home/kaizh/logs/verify_regnet_x_800mf.out <==
Test:  Total time: 0:08:57
Test:  Acc@1 77.522 Acc@5 93.826

==> /data/home/kaizh/logs/verify_regnet_x_8gf.out <==
Test:  Total time: 0:20:10
Test:  Acc@1 81.682 Acc@5 95.678

==> /data/home/kaizh/logs/verify_regnet_y_16gf.out <==
Test:  Total time: 0:22:47
Test:  Acc@1 82.886 Acc@5 96.328

==> /data/home/kaizh/logs/verify_regnet_y_1_6gf.out <==
Test:  Total time: 0:22:04
Test:  Acc@1 80.876 Acc@5 95.444

==> /data/home/kaizh/logs/verify_regnet_y_32gf.out <==
Test:  Total time: 0:20:00
Test:  Acc@1 83.368 Acc@5 96.498

==> /data/home/kaizh/logs/verify_regnet_y_3_2gf.out <==
Test:  Total time: 0:17:23
Test:  Acc@1 81.982 Acc@5 95.972

==> /data/home/kaizh/logs/verify_regnet_y_400mf.out <==
Test:  Total time: 0:12:47
Test:  Acc@1 75.804 Acc@5 92.742

==> /data/home/kaizh/logs/verify_regnet_y_800mf.out <==
Test:  Total time: 0:11:37
Test:  Acc@1 78.828 Acc@5 94.502

==> /data/home/kaizh/logs/verify_regnet_y_8gf.out <==
Test:  Total time: 0:17:12
Test:  Acc@1 82.828 Acc@5 96.330
```